saleor/userprofile/migrations/0013_auto_20171120_0521.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2017-11-20 11:21
+from __future__ import unicode_literals
+
+from django.db import migrations
+import phonenumber_field.modelfields
+import phonenumbers
+
+
+def convert_phone_number_to_phonenumberfield(apps, schema_editor):
+    Address = apps.get_model("userprofile", "Address")
+    for address in Address.objects.all():
+        if address.phone:
+            phone_number = phonenumbers.parse(address.phone.raw_input,
+                                              address.country.code)
+            address.phone = phone_number
+            address.save()
+
+
+def convert_phone_number_to_str(apps, schema_editor):
+    Address = apps.get_model("userprofile", "Address")
+    for address in Address.objects.all():
+        if address.phone:
+            phone_number = str(address.phone)
+            address.phone = phone_number
```

Comment:
This does not do what was requested. Please see previous review comments for an example.